pkgs/tools/archivers/gnutar/default.nix

```diff
@@ -24,33 +30,49 @@ stdenv.mkDerivation rec {
   # gnutar tries to call into gettext between `fork` and `exec`,
   # which is not safe on darwin.
   # see http://article.gmane.org/gmane.os.macosx.fink.devel/21882
-  postPatch = lib.optionalString stdenv.isDarwin ''
+  postPatch = lib.optionals stdenv.isDarwin ''
```

Comment:
This is a mistake, revert

Reply:
I reverted it, however, there are occurences of `postPatch = lib.optionals ` in `nixpkgs`. Maybe we should also fix them.

Reply:
Feel free to go ahead in another PR. 👍